source/partials/_footer.slim

```diff
@@ -29,10 +28,10 @@ footer
           | Community
         .u-xSmallMargin
         ul.SitemapTopic-links
-          li = link_to 'Creators Schools', '/community#creatorsschool', class: 'SitemapTopic-link'
-          li = link_to 'RubyConf Portugal', '/community#rubyconfpt', class: 'SitemapTopic-link'
-          li = link_to 'BragaRB', '/community#bragarb', class: 'SitemapTopic-link'
-          li = link_to 'BragaJS', '/community#bragajs', class: 'SitemapTopic-link'
+          li = link_to 'Mirror Conf', 'http://www.mirrorconf.com/', class: 'SitemapTopic-link'
+          li = link_to 'Braga.Design', 'https://www.meetup.com/bragadesign/', class: 'SitemapTopic-link'
+          li = link_to 'Braga.JS', 'https://www.meetup.com/bragajs/', class: 'SitemapTopic-link'
+          li = link_to 'Braga.Product', 'https://www.meetup.com/Braga-Product/', class: 'SitemapTopic-link'
```

Comment:
If we're adding `target=_blank` on all, can we add the corresponding `rel` attributes? I feel like we should [enforce this practice](https://mathiasbynens.github.io/rel-noopener/)